File: week8/CodeKata8.php
interface QADevOpsInterface
{
    public function doAnApp();
    public function runSomeDockers();
    public function validateTestPlans();
    public function makeNewProject(); 

}

class QADevOps implements QADevOpsInterface
{
    public function doAnApp() {
        return "Done!";
    }

    public function runSomeDockers() {
        return "Done!";
    }

    public function validateTestPlans() {
        return "Done";
    }

    public function makeNewProject() {
        $this->doAnApp();
        $this->runSomeDockers();
        $this->validateTestPlans();
        return "Keep calm and wait for benefits";
    }
}

File: week8/CodeKata8.java
public interface QADevOpsInterface {
    public String doAnApp();
    public String runSomeDockers();
    public String validateTestPlans();
    public String makeNewProject(); 
}

public class QADevOps implements QADevOpsInterface{
    public QADevOps() {
    }

    public String doAnApp() {
        return "Done!";
    }

    public String runSomeDockers() {
        return "Done!";
    }

    public String validateTestPlans() {
        return "Done";
    }

    public String makeNewProject() {
        doAnApp();
        runSomeDockers();
        validateTestPlans();
        return "Keep calm and wait for benefits";
    }
}
